pkg/controllers/job/job_controller_actions.go

```diff
@@ -745,27 +748,29 @@ func (cc *jobcontroller) calcPGMinResources(job *batch.Job) *v1.ResourceList {
 		} else {
 			tp.priority = priorityClass.Value
 		}
-
 		tasksPriority = append(tasksPriority, tp)
 	}
 
 	sort.Sort(tasksPriority)
 
-	minAvailableTasksRes := v1.ResourceList{}
+	minReq := v1.ResourceList{}
 	podCnt := int32(0)
 	for _, task := range tasksPriority {
 		for i := int32(0); i < task.Replicas; i++ {
 			if podCnt >= job.Spec.MinAvailable {
 				break
 			}
+
 			podCnt++
-			for _, c := range task.Template.Spec.Containers {
-				addResourceList(minAvailableTasksRes, c.Resources.Requests, c.Resources.Limits)
+			pod := &v1.Pod{
+				Spec: task.Template.Spec,
 			}
+			res, _ := quotacore.PodUsageFunc(pod, clock.RealClock{})
```

Comment:
Would you calc the minReq with volcano resource calculate functions? It's better not import too much k8s func like PodUsageFunc.

Reply:
to avoid rewrite the functions when the resourcequota logic in k8s is modified..